script/upload

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -x
+
+export PATH=/snap/bin:$PATH
+
+charm whoami
+RET=$?
+if ((RET > 0)); then
+    echo "Not logged into charm store"
+    exit 1
+fi
+
+URL=$(charm push . cs:~"$NAMESPACE"/"$CHARM" | yq r - url)
+IMAGE_NAME="$CHARM-image"
+IMAGE=$(yq r "charms/$CHARM/metadata.yaml" "resources.$IMAGE_NAME.upstream-source")
+docker pull "$IMAGE"
```

Comment:
Do we need to clean this up with something like `docker rmi` after we've attached?

Reply:
Per chat, it's helpful for local dev to leave the image cached, and for CI, we expect the worker environments to be cleaned up anyway, and this is supported by the fact that the current implementation already [pulls images](https://github.com/charmed-kubernetes/jenkins/blob/master/jobs/build-charms/charms.py#L368) and doesn't do anything to clean them up.

Reply:
this feels like a good enough response to me -- if we need to address CI or local dev litter later, we can reassess.